R/load_pkgs.R

```diff
@@ -78,31 +86,50 @@ load_pkgs <- function(branch_name = NULL, git_pull = FALSE, from_commit = list()
       stdout = TRUE
     )
 
-    all_branches <- trimws(sub("origin\\/", "", sub("\\*", "", c(local_pkg_branches, remote_pkg_branches))))
+    all_branches <- trimws(
+      sub(
+        "origin\\/", "",
+        sub("\\*", "", c(local_pkg_branches, remote_pkg_branches))
+      )
+    )
 
     if (!is.null(branch_name)) {
       for (branch in branch_name) {
         if (branch %in% all_branches) {
           cat(
             paste0(
-              "\033[0;92m", branch, " branch exist in ",
-              pkg, "\033[0m\n"
+              "\033[0;92m", branch, " branch exist for \033[0m",
+              "\033[0;94m", pkg, "\033[0m\n"
             )
           )
 
-          system2(
+          checkout_error <- system2(
             "cd",
             args = c(path, paste("&& git checkout", branch)),
             stdout = FALSE,
             stderr = FALSE
           )
+          if (checkout_error) {
+            message <- paste(
+              "\033[0;91m`git checkout` failed.",
+              "Please check your local changes for\033[0;94m",
+              pkg, "\033[0m\n"
```

Comment:
```suggestion
            message <- paste(
              "\033[0;91mCan't switch to", branch, "branch for\033[0;94m",
              pkg, "\033[0m\n",
              " \033[0;91mPlease check your local changes. \033[0m"
            )
```

What do you think about making the message more user friendly by not mentioning `git checkout`?